script.js

```diff
@@ -1,4 +1,88 @@
+var rollDice = function () {
+  var numberRolled = Math.floor(Math.random() * 6) + 1;
+  return numberRolled;
+};
+var player1Scores = [];
+var player2Scores = [];
+var player1Dice1 = 0;
+var player1Dice2 = 0;
+var player2Dice1 = 0;
+var player2Dice2 = 0;
+var player1Total = 0;
+var player2Total = 0;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  //player 1 starts a turn if these conditions are met
+  if (
+    player1Scores.length == player2Scores.length &&
+    player1Dice1 == 0 &&
+    player2Dice1 == 0
+  ) {
+    player1Dice1 = rollDice();
+    player1Dice2 = rollDice();
+    return `Player 1 has rolled ${player1Dice1} and ${player1Dice2}. <br><br>Please select the dice that comes first by submitting '1' or '2'`;
+  } //if player 1 chooses order
+  else if (
+    player1Scores.length == player2Scores.length &&
+    player1Dice1 != 0 &&
+    player2Dice1 == 0
+  ) {
+    if (input != "1" && input != "2") {
+      return 'Please enter "1" or "2"';
+    } else if (input == "1") {
+      var currentPlayerScore = player1Dice1 * 10 + player1Dice2;
+      player1Scores.push(currentPlayerScore);
+      player1Total += currentPlayerScore;
+      return `Player 1 has rolled ${currentPlayerScore}. Please press submit for player 2 to roll`;
+    } else {
+      var currentPlayerScore = player1Dice2 * 10 + player1Dice1;
+      player1Scores.push(currentPlayerScore);
+      player1Total += currentPlayerScore;
+      return `Player 1 has rolled ${currentPlayerScore}. Please press submit for player 2 to roll`;
+    }
```

Comment:
line 30 - 42 could be kept more concise as you have the same code in each conditional block e.g 

```suggestion
    if (input != "1" && input != "2") {
      return 'Please enter "1" or "2"';
    } else if (input == "1") {
      var currentPlayerScore = player1Dice1 * 10 + player1Dice2;
    } else {
      var currentPlayerScore = player1Dice2 * 10 + player1Dice1;
    }
      player1Scores.push(currentPlayerScore);
      player1Total += currentPlayerScore;
      return `Player 1 has rolled ${currentPlayerScore}. Please press submit for player 2 to roll`;
```